department-of-veterans-affairs/va.gov-cms
# VAGOV-4423:

**Synopsis**
- Updates redirects form to allow for match type field processing.

**QA**
- Go to admin/config/search/redirect/edit/233 (or any redirect edit form) and confirm that a match type field is available.
- Change the value of the form, save, and revisit the edit form, confirming that it was saved
- Go to `graphql/explorer` and run this graphql query: 
```
{
  redirectQuery(limit: 10000) {
    entities {
      ... on Redirect {
        type
        statusCode
        created
        redirectSource {
          path
        }
        redirectRedirect {
          options
          title
          uri
          url {
            path
            routed
          }
        }
      }
    }
  }
}
``` 
- Look for the item you edited, and confirm that the value was stored in the `options` segment of the array for this item.
- Repeat the above steps with a new redirect: `/admin/config/search/redirect/add`

**Screenshots**
![Screenshot_2019-07-10_18-05-53](https://user-images.githubusercontent.com/2404547/61008595-f5949a00-a33d-11e9-9405-106082945450.png)
![Screenshot_2019-07-10_18-06-03](https://user-images.githubusercontent.com/2404547/61008596-f5949a00-a33d-11e9-994b-da3afc9cb2f7.png)
